image/distributionutil/login.go

```diff
@@ -41,7 +44,24 @@ func Login(ctx context.Context, authConfig *types.AuthConfig) error {
 	}
 
 	modifiers := dockerRegistry.Headers(dockerversion.DockerUserAgent(ctx), nil)
-	authTransport := transport.NewTransport(dockerRegistry.NewTransport(nil), modifiers...)
+	tlsConfig := tlsconfig.ServerDefault()
+	//skip insecure verify
+	tlsConfig.InsecureSkipVerify = true
```

Comment:
does the following config copy from dockerRegistry.NewTransport(nil)? if it does, we could get transport from dockerRegistry.NewTransport(nil), and change its TLSClientConfig.InsecureSkipVerify.
And is it a good solution to skip insecure verification always?

Reply:
Can the environment variable 'SKIP_TLS_VERIFY=true' be used to determine whether to skip?